app/src/main/java/woowacourse/payments/ui/component/Card.kt

```diff
@@ -17,42 +18,67 @@ import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.shadow
 import androidx.compose.ui.res.colorResource
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.font.FontWeight
 import androidx.compose.ui.text.style.TextOverflow
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import woowacourse.payments.R
+import woowacourse.payments.model.CardVendor
 import woowacourse.payments.ui.allcards.util.CardFormatter
 import woowacourse.payments.ui.theme.AndroidpaymentsTheme
 import woowacourse.payments.ui.uimodel.CardInfoUiModel
 import woowacourse.payments.ui.uimodel.CardInfoUiState
+import woowacourse.payments.ui.uimodel.toUiModel
 
 @Composable
 fun Card(
     cardInfoUiState: CardInfoUiState,
     showCardInfo: Boolean = false,
+    onClick: () -> Unit = {},
 ) {
+    val color =
+        cardInfoUiState.vendor?.let {
+            colorResource(id = it.vendorColorId)
+        } ?: colorResource(id = R.color.payments_card_background)
```

Comment:
```suggestion
    val color = colorResource(cardInfoUiState.vendor?.vendorColorId ?: R.color.payments_card_background)
```
p3) 객체 지정 람다의 장단점은 어떤 것이 있을까요? 취향 차이일 수 있으니 꼭 반영하지 않으셔도 괜찮습니다 :)

Reply:
오오 더 코드가 깔끔해졌네요 ㄷㄷㄷ
객체 지정 람다도 남용하면 몸에 좋지 않군요. 

[refactor: 수신 객체 지정 람다 - >엘비스 연산자로 변경 및 clickable 순서 변경](https://github.com/woowacourse/android-payments/pull/73/commits/a3f4f398a90b6daf40b4beb7df53eb2bd0864666)